Web_Development/FrontEnd/Colors-in-CSS_by_RB.md

```diff
@@ -0,0 +1,9 @@
+# Domain of contribution: FrontEnd with HTML , CSS and JS
+# The Topic : Colors in CSS
+## Issue number that this fixes: #770
+## The Link of the Google Colab for the Documentation :
+   [Colors in CSS by RB.ipnb](https://colab.research.google.com/drive/1HgkiPo41jomjcgWgRXEmVdabrtUWqLz-?usp=sharing)
```

Comment:
Add the content in this file itself, don't give a link to another file unless it is an audio or a video.

Reply:
Sorry but I think in the GWOC Contributions guidelines it was mentioned that the contributors have to submit work by only in Google Collab . Check it please.